### PR TITLE
Add bot

### PR DESCRIPTION
**Pull request by endersonmenezes**
I've already organized some hackathons here in Brazil, and this bot has always been useful to me to account for members' participation in a channel.

### CODE REVIEW

**Comment by juanpflores** [comment]
Hi @endersonmenezes 👋🏼 
Would love to add this bot to the list! Could you solve the merge conflicts so we can merge it?

**Review by juanpflores — CHANGES_REQUESTED** [review]
Merge conflicts need to be solved before merging

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Now that you're all set up with a Student Dev Pack, have a Discord account, and
 | Bot | Description |
 |-|-|
 | [Zira.gg](https://zira.gg) | Zira lets users add and remove roles for themselves by simply reacting to a message. |
+| [Suivix](https://suivix.xyz/en) | You are a professor or maybe even a business manager and you want to take attendance on Discord. No worries! Suivix will take care of it for you. |
 | [Templae](https://github.com) | Template |
```
Comment (XynoxTheDev) [inline README.md:51]: What's this is for?

**Comment by XynoxTheDev** [comment]
@juanpflores There is no response in this PR and it has been a week, ig this PR should be closed and contributors can request to open it in the future!

File: README.md (hunk 1 above)
Comment (endersonmenezes) [reply README.md:51]: Sorry i'm late. Its a bot for discord.

**Comment by endersonmenezes** [comment]
You can close it, I'll see how the model is now and try to contribute.

**Pull request closed without merging**